src/views/admin/AllOperations.vue

```diff
@@ -0,0 +1,114 @@
+        <button v-if="!gotOps" class="btn btn-blue-primary p-4 mt-10" @click="requestData">Request Operations</button>
+        <div v-else class="w-full flex flex-col">
+            <search-bar v-model:message="message" @refresh="refresh" />
+            <dashboard-component
+                identifier="archive"
+                class="w-full mt-5"
+                :operations="filteredOperations"
+                title="Archived Operations"
+                :is-archive="true"
+            />
+        </div>
+    </div>
+</template>
+<script lang="ts">
+    import { useStore } from "@/use/store/store";
+    import { ref, onBeforeMount, computed } from "vue";
+    import LoadingSpinner from "@/components/common/loading/Spinner.vue";
+    import { checkPrivilege } from "@/use/helpers/PermissionHelper";
+    import DashboardComponent from "@/components/common/dashboard/DashboardComponent.vue";
+    import Operation, { ApprovalList } from "@/api/api_models/operation_management/Operation";
+    import { UC4Identifier } from "@/api/helpers/UC4Identifier";
+    import { OperationStatus } from "@/api/api_models/operation_management/OperationState";
+    import Router from "@/use/router/";
+    import OperationManagement from "@/api/OperationManagement";
+    import GenericResponseHandler from "@/use/helpers/GenericResponseHandler";
+    import SearchBar from "@/components/common/SearchBar.vue";
+    import { MutationTypes } from "@/use/store/mutation-types";
+    import filterOperations from "@/use/helpers/filterOperations";
+
+    export default {
+        name: "AllOperationsPage",
+        components: {
+            LoadingSpinner,
+            DashboardComponent,
+            SearchBar,
+        },
+
+        setup() {
+            const busy = ref(false);
+            const gotOps = ref(false);
+            const operations = ref([] as Operation[]);
+            const watchedOperations = ref([] as Operation[]);
+            const message = ref("");
+
+            const filteredOperations = computed(() => {
+                return filterOperations(operations.value, message.value);
+            });
+
+            async function requestData() {
+                busy.value = true;
+                await getOperations();
+                busy.value = false;
+            }
+
+            async function getOperations() {
+                const operationManagement = new OperationManagement();
+                const handler = new GenericResponseHandler("operations");
+                const response = await operationManagement.getOperations(undefined, undefined, undefined, false);
+                const result = handler.handleResponse(response);
+
+                //Show empty list if no results given
+                operations.value = result;
+                gotOps.value = true;
+                getWatchedOperations();
+            }
+
+            async function getWatchedOperations() {
+                //TODO API CALL
```

Comment:
watched operations is already implemented in the API via getOperations(........  , true)

Reply:
Yes, I know, that's how I already get pending and finished operations. This is a relict from my ideas for tagging already watched operations. I can remove this if you want me to, but in some way, this function will be used :)

Reply:
I added the API call in https://github.com/upb-uc4/ui-web/pull/785/commits/da722d0a46b3532d16f10acebc5c1abed722635b